ext/standard/tests/filters/strip_tags.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+strip_tags filter - ctor and dtor
+--CREDIT--
+H Hatfield hatfieldje@gmail.com UPHPU Test Fest
+--FILE--
+<?php
+# vim600:syn=php:
```

Comment:
```suggestion
```

Reply:
This one is the only change needed?

Reply:
This is copy/paste from whichever test I used as a template. It seems to show up in quite a few files. I have no strong ties to this line, but would like to understand why it should be removed.

Reply:
@hhatfield These are so called Vim modes. They are long gone from php-src in favor of the EditorConfig and manual user side configurations. See this change:
https://github.com/php/php-src/commit/92ac598aabd336593a47ed3959f1031674b763e6

and here https://externals.io/message/101206

Yes, they were present once in some of those files. Some are still there but only on very rare places (for eample *.w32 files to enable syntax highlighting and for some documentation purposes). Long term: they should all be replaced with different approaches. For example, EditorConfig once more options will be available.